src/docs/sdk/performance/span-operations.mdx

```diff
@@ -0,0 +1,125 @@
+|            | resource.img    |                                                                                                                     |
+| browser    |                 | Usage of browser APIs or functionality                                                                              |
+|            | browser.paint   |                                                                                                                     |
+| mark       |                 | Usage of [performance.mark() API](https://developer.mozilla.org/en-US/docs/Web/API/Performance/mark)                |
+| measure    |                 | Usage of [performance.measure() API](https://developer.mozilla.org/en-US/docs/Web/API/Performance/measure)          |
+| user       |                 |                                                                                                                     |
+| animate    |                 |                                                                                                                     |
+
+### JS Frameworks
+
+| Op      | Usage        | Description                                    |
+| ------- | ------------ | ---------------------------------------------- |
+| react   |              | Spans related to [React](https://reactjs.org/) |
+|         | react.mount  |                                                |
+|         | react.render |                                                |
+| vue     |              |                                                |
+|         | vue.mount    |                                                |
+|         | vue.update   |                                                |
+| angular |              |                                                |
+| ember   |              |                                                |
+
+## Web Server
+
+| Op        | Usage           | Description |
+| --------- | --------------- | ----------- |
+| http      |                 |             |
+|           | http.client     |             |
+|           | http.server     |             |
+| rpc       |                 |             |
+| grpc      |                 |             |
+| template  |                 |             |
+|           | template.init   |             |
+|           | template.parse  |             |
+|           | template.render |             |
+| view      |                 |             |
+|           | view.render     |             |
+| serialize |                 |             |
+
+## Web Frameworks
+
+| Op      | Usage             | Description |
+| ------- | ----------------- | ----------- |
+| django  |                   |             |
+|         | django.middleware |             |
+|         | django.view       |             |
+| express |                   |             |
+| rails   |                   |             |
+| rack    |                   |             |
+| console |                   |             |
```

Comment:
grabbed from https://github.com/getsentry/sentry-symfony/blob/5129f962a9b921f08fa2100fc5593cd6d8f06566/src/EventListener/TracingConsoleListener.php#L62, but this seems to be more related to running stuff on the command line. I'll move into web server (since using rails, django cmd line is pretty common operations).